sdk/version/version_base.go

```diff
@@ -8,7 +8,7 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
-	Version           = "1.5.8"
-	VersionPrerelease = ""
+	Version           = "1.5.9"
+	VersionPrerelease = "rc"
```

Comment:
This is not an RC. Why is this set?

Reply:
Oh, do minor releases not have release candidates? Should I just set it back to `""` ?

Reply:
I don't think so. In my opinion, yes, it should be set to "".

Reply:
Sounds good!